libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1370,23 +1370,31 @@ string YulUtilFunctions::storageByteArrayPopFunction(ArrayType const& _type)
 	});
 }
 
-string YulUtilFunctions::storageArrayPushFunction(ArrayType const& _type)
+string YulUtilFunctions::storageArrayPushFunction(ArrayType const& _type, TypePointer _fromType)
```

Comment:
```suggestion
string YulUtilFunctions::storageArrayPushFunction(ArrayType const& _type, Type const* _fromType)
```
(since we don't use shared_ptr here anymore, there is no reason to have a named type)